src/api/app/views/webui2/webui/monitor/_workers_table.html.haml

```diff
@@ -1 +1,48 @@
-%h2 Workers
+%h2.nowrap#workers-title
+  Workers
+  %span.hidden#workers-updating (updating...)
+
+.col-12
+  %form.float-sm-right{ action: '/' }
+    %label Show:
+    - worker_display_options = [['Package name', 'package'], ['Project', 'project'], ['Repository', 'repository'], ['Architecture', 'arch']]
+    = select_tag(:worker_display, options_for_select(worker_display_options, 'package'), id: 'worker-display', class: 'custom-select')
+
+#workers{ data: { monitor_path: "#{url_for(controller: :monitor, action: :update_building)}",
```

Comment:
`url_for(...)` can be replaced by a route helper: `monitor_update_building_path`

Reply:
done